etc/env/example.env

```diff
@@ -85,7 +85,7 @@ MINIO_SELF_CERT=0
 HTSGET_VERSION=v4.1.0
 HTSGET_PRIVATE_URL=http://htsget:3000
 HTSGET_PUBLIC_URL=${TYK_LOGIN_TARGET_URL}/${TYK_HTSGET_API_LISTEN_PATH}
-HTSGET_PORT=3333
+HTSGET_PORT=3000
```

Comment:
Why are you proposing to change these ports everywhere, instead of just in the C3G .env file?